src/Log/EmptyPsrLoggerAdapter.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mollie\WooCommerce\Log;
+
+use Psr\Log\AbstractLogger;
+
+class EmptyPsrLoggerAdapter extends AbstractLogger
```

Comment:
Maybe you can use https://github.com/php-fig/log/blob/master/src/NullLogger.php instead?